test/simplegraphs/generators/randgraphs.jl

```diff
@@ -384,4 +384,27 @@
         @test μ1 - sv1 <= 0.3 * 5 <= μ1 + sv1 # since the stdev of μ1 is around sv1/sqrt(N), this should rarely fail
         @test μ2 - sv2 <= 0.7 * 3 <= μ2 + sv2
     end
+
+    @testset "bernoulli graphs" begin
+        n = 50
+        Λ = rand(n, n)
+        ρ = 1.0 # isomorphism case
```

Comment:
If I understand correctly, it is a bit more than ismorphism here, as the graphs are actually equal without shuffling the vertices.